lib/home/screens/home_options.dart

```diff
@@ -1,14 +1,30 @@
+import 'package:example_whit_github/widgets/text_fields/custom_card_field.dart';
 import 'package:flutter/animation.dart';
+import 'package:flutter/material.dart';
 
 
 
-class
-
 class HomeOptions extends StatelessWidget {
   const HomeOptions({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Scaffold(
+      body: ListView(
+        padding: EdgeInsets.symmetric(horizontal: 10,vertical: 10),
+        children: [
+          CustomCard(
+              ImagenPageCard:
+              Image(image: NetworkImage("https://w7.pngwing.com/pngs/369"
```

Comment:
what happen if we put a diferent size of image?

Reply:
i don't now